.github/workflows/release-drafter.yml

```diff
@@ -18,14 +18,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_PR: ${{ github.event.pull_request.number }}
-#  update_release_draft:
-#    runs-on: ubuntu-latest
-#    permissions:
-#      contents: write
-#      pull-requests: read
-#    steps:
-#      - uses: release-drafter/release-drafter@v6
-#        with:
-#          config-name: release-drafter.yml # located in .github/ in default branch
-#        env:
-#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  update_release_draft:
+    runs-on: ubuntu-24.04
```

Comment:
@secretsayan I think we shouldn't change this to `ubuntu-24.04` at all places (except ORCA CI). This is specifically needed when Gitlab CI running PHPUnit tests, because the `ubuntu-latest` currently resolves to `ubuntu-22.04` and it uses old version of sqlite which Drupal 11 doesn't support.

Reply:
Done